custom/icds_reports/reports/awc_reports.py

```diff
@@ -23,7 +23,7 @@
     person_has_aadhaar_column, person_is_beneficiary_column, get_status, wasting_moderate_column, \
     wasting_severe_column, stunting_moderate_column, stunting_severe_column, current_month_stunting_column, \
     current_month_wasting_column, hfa_recorded_in_month_column, wfh_recorded_in_month_column, \
-    chosen_filters_to_labels, default_age_interval, get_anemic_status, get_symptoms, get_counseling, get_tt_dates, is_anemic
```

Comment:
Nit: given how long this import is, it might actually be nicer if you were to do:
``` from custom.icds_reports import utils```
then reference utils.{func}.